sqlserver/datadog_checks/sqlserver/metrics.py

```diff
@@ -473,6 +473,82 @@ def fetch_metric(self, rows, columns):
             self.report_function(metric_name, column_val, tags=metric_tags)
 
 
+# https://docs.microsoft.com/en-us/sql/relational-databases/system-tables/backupset-transact-sql?view=sql-server-ver15
```

Comment:
Include a small snippet, microsoft tends to change their documentation urls and links become obsolete